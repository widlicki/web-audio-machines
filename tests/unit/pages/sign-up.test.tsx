import { render } from '@testing-library/react'
import SignUp from '../../../pages/sign-up'

describe('Sign Up', () => {
  it('renders sign-up page according to snapshot', () => {
    const { container } = render(<SignUp />)
    expect(container).toMatchSnapshot()
  })
})
