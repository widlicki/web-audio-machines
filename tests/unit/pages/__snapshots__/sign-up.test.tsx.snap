// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sign Up renders sign-up page according to snapshot 1`] = `
<div>
  <header
    class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation0 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionFixed mui-fixed css-xnb1ep-MuiPaper-root-MuiAppBar-root"
  >
    <div
      class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-1wlywpi-MuiToolbar-root"
    >
      <div
        class="MuiBox-root css-1rr4qq7"
      />
      <a
        class="MuiTypography-root MuiTypography-h4 MuiLink-root MuiLink-underlineAlways css-l2y2e4-MuiTypography-root-MuiLink-root"
        href="/"
      >
        web • audio • machines
      </a>
      <div
        class="MuiBox-root css-1j9bsaa"
      >
        <a
          class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-kq33hk-MuiTypography-root-MuiLink-root"
        >
          Sign In
        </a>
        <a
          class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-kq33hk-MuiTypography-root-MuiLink-root"
        >
          Sign Up
        </a>
      </div>
    </div>
  </header>
  <div
    class="MuiContainer-root css-l1bhnm-MuiContainer-root"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-4jjtl3-MuiContainer-root"
    >
      <div
        class="MuiBox-root css-1fc829"
      >
        Sign Up
      </div>
    </div>
  </div>
  <div
    class="MuiContainer-root css-1s55f6l-MuiContainer-root"
  >
    <div
      class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-12tubgu-MuiGrid-root"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
        >
          © web audio machines 
          2022
        </p>
      </div>
    </div>
  </div>
</div>
`;
